src/data.toml

```diff
@@ -17,4 +17,4 @@ addon_404 = "Please do not visit the URL directly since it won't display anythin
 cloud_sync_not_working = "You might be using an outdated version of Cloud Sync.\n\n- Ensure you have the [latest version](<https://bunny.nexpid.xyz/cloud-sync/>) installed.\n- Reinstall the [antied plugin](<https://angelix1.github.io/MP/angel/antied/>) if you use it.\n\nPerform a local backup to avoid losing data permanently."
 android_15_error = "Refer to https://discord.com/channels/1196075698301968455/1205101050739167272/1264529990997905442"
 fake_profile_theme_and_effects_guide = "## How to Use the Fake Profile Themes and Effects Plugin\n\n1. Select the primary color.\n2. Select the accent color.\n3. Select an effect.\n4. Copy FPTE.\n5. Paste what you just copied into the \"About Me\" section, starting on a new line at the end.\n\n**Note:** You *might* experience issues when using the Gboard clipboard to paste the FPTE. Instead, use the hold-to-paste method.\n\n![Example](https://imgur.com/paPTG3J)"
-free_nitro = "## List of Plugins that Provide **Nitro-Like Features**:\n1. [Freemoji Plugin](<https://rico040.github.io/bunny-plugins/freemoji/>)\n2. [Realmoji Plugin](<https://redstonekasi.github.io/vendetta-plugins/realmoji/>)\n3. [Fake Profile Themes and Effects Plugin](<https://ryan-0324.github.io/vendetta-plugins/FakeProfileThemesAndEffects/>)\n4. [UserBG Plugin](<https://rico040.github.io/bunny-plugins/userbg/>)\n5. [UserPFP](<https://bunny.nexpid.xyz/usrpfp/>)\n6. [Free Stickers Plugin](<https://aliernfrog.github.io/vd-plugins/FreeStickers/>)\n7. [Nitro Themes Plugin](<https://nullpo1nt3d.github.io/VendettaPlugins/NitroThemes/>)"
+free_nitro = "## List of Plugins that Provide **Nitro-Like Features**:\n1. [Freemoji Plugin](<https://rico040.github.io/bunny-plugins/freemoji/>)\n2. [Realmoji Plugin](<https://redstonekasi.github.io/vendetta-plugins/realmoji/>)\n3. [Fake Profile Themes and Effects Plugin](<https://ryan-0324.github.io/vendetta-plugins/FakeProfileThemesAndEffects/>)\n4. [UserBG Plugin](<https://rico040.github.io/bunny-plugins/userbg/>)\n5. [UserPFP plugin](<https://bunny.nexpid.xyz/usrpfp/>)\n6. [Free Stickers Plugin](<https://aliernfrog.github.io/vd-plugins/FreeStickers/>)\n7. [Nitro Themes Plugin](<https://nullpo1nt3d.github.io/VendettaPlugins/NitroThemes/>)"
```

Comment:
I would remove the "plugin" from the names as this is a list of plugins already, it's implicit

Reply:
Alright, removing "plugin" phrase from all of the hyperlink texts.